src/Amadevus.RecordGenerator.Generators/EqualityGenerators.cs

```diff
@@ -154,14 +154,14 @@ private static MemberDeclarationSyntax GenerateGetHashCode(RecordDescriptor desc
         {
             const string varKeyWord = "var";
             const string hashCodeVariableName = "hashCode";
+
             const int hashCodeInitialValue = 2085527896;
             const int hashCodeMultiplicationValue = 1521134295;
-            // public override int GetHashCode() {
-            //   var hashCode = 2085527896;
-            //   hashCode = hashCode * -1521134295 + EqualityComparer<TProp>.Default.GetHashCode(Prop);
-            //   ...
-            //   return hashCode;
-            // }
+
+            StatementSyntax expression = descriptor.Entries.Length == 1
```

Comment:
This can be inlined, it's short.

If not, use var and change the name to `statement` or sth else, but it's definitely not an `expression` :)

Reply:
My preference would be to not inline this sort of code as it becomes hidden in the syntax tree and therefore more difficult to read.  All of these variables will be compiled out so not sure the value in making the code so tight.

Reply:
It's an interesting observation. Noted.